README.md

```diff
@@ -0,0 +1,12 @@
+
+## helper function demo
+Currently, this repo is designed to work with Flakegaurd out of the box. However, the following demo below shows the included helper function in action to convert the data for non flake-part Flakegaurd users.
+
+```nix
+networking.wireguard.interfaces.wg0 = let
+ peers = inputs.automous-zones.flakeModules.asluni.wireguard.networks.asluni.peers.by-name;
+ azlib = inputs.automous-zones.lib;
+in {
+    peers = azlib peers; 
+};
+```
```

Comment:
the directive `networking.wireguard.interfaces.wg0` with respect to `wg0`, can be labeled as anything, but for sake of clarity changing it to `asluni`

`azlib` should instead be `aslib`, automous-zones is was shorted from "automous system numbers" in bgp.

finally, the example should provide all the information needed to recomplete.

in this case, the flake implementation, and the missing information in `networking.wireguard.interfaces.wg0 (-> asluni)`